src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -24,6 +24,44 @@
 
 namespace Tizen.Multimedia
 {
+    /// <summary>
+    /// Represents properties for streaming buffering time
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public struct StreamingBufferingTime
+    {
+        /// <summary>
+        /// Initializes a new instance of the StreamingBufferingTime struct.
+        /// </summary>
+        /// <param name="preBuffMs">The buffer time to start playback.</param>
+        /// <param name="reBuffMs">The buffer time during playback if player enter pause state for buffering.</param>
+        /// <since_tizen> 5 </since_tizen>
+        public StreamingBufferingTime(int preBuffMs, int reBuffMs)
+        {
+            PreBuffMs = preBuffMs;
+            ReBuffMs = reBuffMs;
+        }
+
+        /// <summary>
+        /// Gets or sets the buffer time to start playback.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public int PreBuffMs
```

Comment:
How about consider below guideline? 
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions#using-abbreviations-and-acronyms
And make it more clearier.

Reply:
I've changed it to PreBufferMs and enhanced the description.